### PR TITLE
Open PR

### PR DESCRIPTION
**Pull request by kelum-p**

### CODE REVIEW

File: adddel.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
 Nested functions, as introduced in Nested Functions, are a convenient means of naming and defining self-contained blocks of code as part of a larger function. However, it is sometimes useful to write shorter versions of function-like constructs without a full declaration and name. This is particularly true when you work with functions or methods that take functions as one or more of their arguments.
```
Comment (Coocoolaka) [inline adddel.md]: blaaathd

File: adddel.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
...
 
-Closure expressions are a way to write inline closures in a brief, focused syntax. Closure expressions provide several syntax optimizations for writing closures in a shortened form without loss of clarity or intent. The closure expression examples below illustrate these optimizations by refining a single example of the sort(_:) method over several iterations, each of which expresses the same functionality in a more succinct way.
-
```
Comment (Coocoolaka) [inline adddel.md]: Comment on a `delete`

File: adddel.md [hunk 3]
```diff
@@ -1,5 +1,3 @@
...
-
+Because the sorting closure is passed as an argument to a method, Swift can infer the types of its parameters and the type of the value it returns. The sort(_:) method is being called on an array of strings, so its argument must be a function of type (String, String) -> Bool. This means that the (String, String) and Bool types do not need to be written as part of the closure expression’s definition. Because all of the types can be inferred, the return arrow (->) and the parentheses around the names of the parameters can also be omitted:
```
Comment (Coocoolaka) [inline adddel.md]: comment on an add

File: adddel.md (hunk 1 above)
Comment (Coocoolaka) [inline adddel.md]: This is a line comment to `blaaathd`

File: addition.md [hunk 4]
```diff
@@ -17,3 +17,7 @@ swapTwoInts(&someInt, &anotherInt)
 print("someInt is now \(someInt), and anotherInt is now \(anotherInt)")
 // Prints "someInt is now 107, and anotherInt is now 3
 ```
+
+If the first string (s1) is greater than the second string (s2), the backwards(_:_:) function will return true, indicating that s1 should appear before s2 in the sorted array. For characters in strings, “greater than” means “appears later in the alphabet than”. This means that the letter "B" is “greater than” the letter "A", and the string "Tom" is greater than the string "Tim". This gives a reverse alphabetical sort, with "Barry" being placed before "Alex", and so on.
```
Comment (Coocoolaka) [inline addition.md]: from borbase

File: one/two/three/four/deep.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Deeply nested file
```
Comment (Coocoolaka) [inline one/two/three/four/deep.md]: Deeply nested from borbase

File: adddel.md (hunk 3 above)
Comment (Coocoolaka) [inline adddel.md]: comment borbase

File: rename.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+### Edge cases
```
Comment (Coocoolaka) [inline rename.md]: edge

File: addition.md [hunk 7]
```diff
@@ -17,3 +17,7 @@ swapTwoInts(&someInt, &anotherInt)
...
+
+However, this is a rather long-winded way to write what is essentially a single-expression function (a > b). In this example, it would be preferable to write the sorting closure inline, using closure expression syntax.
```
Comment (Coocoolaka) [inline addition.md]: this is cooooooo

File: deletion.md [hunk 8]
```diff
@@ -1,5 +1,3 @@
 A string is a series of characters, such as "hello, world" or "albatross". Swift strings are represented by the String type. The contents of a String can be accessed in various ways, including as a collection of Character values.
```
Comment (Coocoolaka) [inline deletion.md]: say

File: deletion.md [hunk 9]
```diff
@@ -1,5 +1,3 @@
...
 
 Swift’s String and Character types provide a fast, Unicode-compliant way to work with text in your code. The syntax for string creation and manipulation is lightweight and readable, with a string literal syntax that is similar to C. String concatenation is as simple as combining two strings with the + operator, and string mutability is managed by choosing between a constant or a variable, just like any other value in Swift. You can also use strings to insert constants, variables, literals, and expressions into longer strings, in a process known as string interpolation. This makes it easy to create custom string values for display, storage, and printing.
-
-Despite this simplicity of syntax, Swift’s String type is a fast, modern string implementation. Every string is composed of encoding-independent Unicode characters, and provides support for accessing those characters in various Unicode representations.
```
Comment (Coocoolaka) [inline deletion.md]: yayaya

Comment (Coocoolaka) [inline deletion.md]: one

File: rename.md (hunk 6 above)
Comment (Coocoolaka) [inline rename.md]: edge 2

Comment (Coocoolaka) [inline rename.md]: edge 3

File: README.md [hunk 10]
```diff
@@ -0,0 +1 @@
+# prtest
```
Comment (Coocoolaka) [inline README.md]: first

File: adddel.md [hunk 11]
```diff
@@ -1,5 +1,3 @@
 Nested functions, as introduced in Nested Functions, are a convenient means of naming and defining self-contained blocks of code as part of a larger function. However, it is sometimes useful to write shorter versions of function-like constructs without a full declaration and name. This is particularly true when you work with functions or methods that take functions as one or more of their arguments.
```
Comment (Coocoolaka) [inline adddel.md]: second

File: one/two/three/four/deep.md (hunk 5 above)
Comment (Coocoolaka) [inline one/two/three/four/deep.md]: deep

File: one/two/three/four/deep.md [hunk 12]
```diff
@@ -0,0 +1,3 @@
...
+to see how to and from files
+add
```
Comment (Coocoolaka) [inline one/two/three/four/deep.md]: add

File: rename.md [hunk 13]
```diff
@@ -0,0 +1,8 @@
...
+
+Does GitHub.com pases these stuff correctly? Dos BorBase?
+
+This is just a rename
```
Comment (Coocoolaka) [inline rename.md]: h

File: rename.md [hunk 14]
```diff
@@ -0,0 +1,8 @@
...
+
+
+
```
Comment (Coocoolaka) [inline rename.md]: n

File: addition.md [hunk 15]
```diff
@@ -17,3 +17,7 @@ swapTwoInts(&someInt, &anotherInt)
 print("someInt is now \(someInt), and anotherInt is now \(anotherInt)")
 // Prints "someInt is now 107, and anotherInt is now 3
 ```
```
Comment (Coocoolaka) [inline addition.md]: quick

File: addition.md (hunk 7 above)
Comment (Coocoolaka) [inline addition.md]: coo

Comment (Coocoolaka) [inline addition.md]: keel

File: deletion.md [hunk 16]
```diff
@@ -1,5 +1,3 @@
 A string is a series of characters, such as "hello, world" or "albatross". Swift strings are represented by the String type. The contents of a String can be accessed in various ways, including as a collection of Character values.
 
 Swift’s String and Character types provide a fast, Unicode-compliant way to work with text in your code. The syntax for string creation and manipulation is lightweight and readable, with a string literal syntax that is similar to C. String concatenation is as simple as combining two strings with the + operator, and string mutability is managed by choosing between a constant or a variable, just like any other value in Swift. You can also use strings to insert constants, variables, literals, and expressions into longer strings, in a process known as string interpolation. This makes it easy to create custom string values for display, storage, and printing.
```
Comment (Coocoolaka) [inline deletion.md]: another

File: rename.md (hunk 6 above)
Comment (Coocoolaka) [inline rename.md]: new comment

File: addition.md [hunk 17]
```diff
@@ -17,3 +17,7 @@ swapTwoInts(&someInt, &anotherInt)
...
+
```
Comment (Coocoolaka) [inline addition.md]: New comment from borbase

File: deletion.md (hunk 8 above)
Comment (kelum-p) [inline deletion.md:2]: Does this appear with a new network call? Edited

File: rename.md [hunk 18]
```diff
@@ -0,0 +1,8 @@
+### Edge cases
+
+Does GitHub.com pases these stuff correctly? Dos BorBase?
+
```
Comment (Coocoolaka) [inline rename.md]: now

Comment (Coocoolaka) [inline rename.md]: now 1

File: adddel.md (hunk 1 above)
Comment (kelum-p) [inline adddel.md:2]: New comment code

h1

# h2

```
code block
```

**string**

_italic_

Comment (Coocoolaka) [inline adddel.md]: Another new comment

File: deletion.md (hunk 8 above)
Comment (Coocoolaka) [inline deletion.md]: second

File: adddel.md (hunk 1 above)
Comment (kelum-p) [inline adddel.md:2]: Reply to new comment

Comment (kelum-p) [inline adddel.md:2]: New markdown

### yay

File: addition.md (hunk 4 above)
Comment (kelum-p) [inline addition.md:21]: This is a very long comment to test weather the label only shows a single line

**Pull request merged**